PhysicsTools/SelectorUtils/src/classes_def.xml

```diff
@@ -9,4 +9,5 @@
   <class name="MuonVPlusJetsIDSelectionFunctor"/>
   <class name="PVSelector"/>
   <class name="RunLumiSelector"/>
+  <class pattern="VersionedSelector<*>"/>
```

Comment:
The `classes_def.xml` should list all types explicitly (we had not so long ago a campaign to remove all patterns).